confidant/schema/blind_credentials.py

```diff
@@ -0,0 +1,71 @@
+import attr
+import toastedmarshmallow
+from marshmallow import fields
+
+from confidant.schema.auto_build_schema import AutobuildSchema
+
+
+@attr.s
+class BlindCredentialResponse(object):
+    id = attr.ib()
+    name = attr.ib()
+    cipher_version = attr.ib()
+    cipher_type = attr.ib()
+    data_key = attr.ib()
```

Comment:
New to `attr`, but should we add a default for `data_key` in case the flag doesn't get passed in with `true`?  The others `credential_keys` and `credential_pairs` include defaults.

Reply:
Ah. Good catch. I just checked the model and data_key is a dict for blind_credentials, so yeah, this should have a dict default. Let me fix that.